### DISCUSSION TITLE
Using prestashop, Symfony Dependency Injection Component creates files in cache as root user

### DISCUSSION THREAD

**Discussion by qualityangel** [symfony/symfony#45438] [Q&A]
### Symfony version(s) affected

3.4.49

### Description

I'm using the latest prestashop, and it use symphony, but Symfony Dependency Injection Component is creating files in the var/cache folder that has root:root user and not user:pascln. This way the website is randomly not working right.

Searched the internet, contacted prestashop but nothing.

### How to reproduce

Install prestashop. look in the cache folder

**Comment by michaoj** [comment]
Hard to help you as we don't know what's your environment.

Do you use docker as your local environment? I that case, maybe the problem relies in the docker_compose file that mounts the var cache folder

**Reply by qualityangel** [reply]
I'm using a clean install, no docker envoirement
